topics/wallet-standard.md

```diff
@@ -0,0 +1,248 @@
+# Wallet Standard
+
+## Messages
+
+
+
+### Permission
+
+The purpose of the permission messages is to establish a connection between a relying party and a wallet.
+
+#### Types
+
+- `text`: A plain `string` value.
+- `blob`: A `string` value describing binary data encoded in base64.
+
+#### Request
+
+`version` (`text`): The version of the standard used.
+
+`senderId` (`text`): The ID of the sender. This should be used to identify the sender of the message.
```

Comment:
How is this different from `sender`? And what is the use-case for it?

I think a use-case example would help here. 😉  Also, if possible, another name might help too, since it is very similar to `sender` but describes something else?

Reply:
Since we agreed the`senderId` didn't belong here, it has been removed